app/code/Magento/SampleData/Console/Command/SampleDataDeployCommand.php

```diff
@@ -79,15 +86,42 @@ protected function configure()
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int|void
+     * @throws FileSystemException
+     * @throws LocalizedException
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $rootJson = json_decode($this->filesystem->getDirectoryRead(DirectoryList::ROOT)->readFile("composer.json"));
+        $rootJson = json_decode(
+            $this->filesystem->getDirectoryRead(
+                DirectoryList::ROOT
+            )->readFile("composer.json")
+        );
```

Comment:
Could you use Json Serializer from Magento package?

Reply:
Hello @lbajsarowicz 
Thank you for your review.
I fixed it. Please check.
Thank you in advance.